### ISSUE TITLE
Define Dashboard Elements

### ISSUE THREAD

**Issue by StrangeUSB** [gophish/gophish#90]
The dashboard has a lot going on, adding some definition of elements could benefit it.

Adding wells is an example:
![dashboard_wells](https://cloud.githubusercontent.com/assets/6924235/12470907/143cf806-bfbe-11e5-834e-13061772c778.png)

Thoughts?

**Comment by jordan-wright** [comment]
I don't mind the idea of a well, as long as it's not too dark. I think it's a great idea and helps separate out the elements in the dashboard.

**Comment by StrangeUSB** [comment]
Yeah I agree, I'm not quite a fan of how the default wells look on it, but it's just something to throw out there.  I think this could help on the campaign results page also.

**Comment by emixam3** [comment]
Hi,
I think dashboards would be more efficients with a time range selection.
For example, I want to filter all the campaigns I launched in 2020, and see related dashboards and statistics.

**Comment by glennzw** [comment]
Good idea. At some point we want to overhaul the entire UI with something more modern, see: https://github.com/gophish/gophish/pull/1565#issuecomment-526181253
